### ISSUE TITLE
Not Achieving Same Accuracy Results

### ISSUE THREAD

**Issue by mariacioffi** [gabrielhuang/reptile-pytorch#7]
Hello, I'm running this code from my local machine. It's a direct fork from this repo. The only updates I've made are to make it compatible with Python3. Are you certain you're seeing ~95% accuracy when running with these hyperparams?:
 --classes 5 --shots 5 --train-shots 10 --meta-iterations 100000 --iterations 5 --test-iterations 50 --batch 10 --meta-lr 0.2 --lr 0.001

Because I am not. Here is my final log output: 
Meta-train
average metaloss 0.9020509621277452
average accuracy 0.7628000104278326

Meta-val
average metaloss 0.9079857150167226
average accuracy 0.7575000119954347

Any ideas why I may be seeing much lower accuracies? Any input would be appreciated. Thanks.

**Comment by gabrielhuang** [comment]
Sorry, it's been a while. Ideally try to run on Python2, there are a few
subtleties that might have broken during conversion. You might also want to
try lower meta-lr/lr
